source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/GenericImageLoaderAsync.java

```diff
@@ -24,11 +24,13 @@
     String m_imageBaseUrl;
     IOnlineImageLoader m_onlineImageLoader = null;
     IDataUriImageLoader m_dataUriImageLoader = null;
+    int m_maxWidth;
 
-    GenericImageLoaderAsync(RenderedAdaptiveCard renderedCard, String imageBaseUrl)
+    GenericImageLoaderAsync(RenderedAdaptiveCard renderedCard, String imageBaseUrl, int maxWidth)
```

Comment:
> GenericImageLoaderAsync(RenderedAdaptiveCard renderedCard, String imageBaseUrl, int maxWidth) [](start = 3, length = 94)

keep the original constructor and add a flag or set m_maxWidth to -1 to know when was the maxWidth actually defined

Reply:
Done.